types/amazon-cognito-auth-js/index.d.ts

```diff
@@ -573,5 +590,65 @@ export class StorageHelper {
      * This is used to return the storage
      * @returns the storage
      */
-    getStorage(): Storage;
+    getStorage(): CognitoAuthStorage;
+}
+
+export interface CognitoAuthStorage extends Pick<Storage, 'setItem' | 'getItem' | 'removeItem' | 'clear'> {}
+
+export interface CookieStorageOptions {
+    /**
+     * Cookies domain (mandatory).
```

Comment:
```suggestion
     * Cookies domain
```

nit: Probably not necessary, given the type enforces its mandatory-ness